make/dev.mk

```
@@ -27,3 +29,12 @@ endif
 	kubectl set image deployment cluster-api-runtime-extensions-nutanix webhook=ko.local/cluster-api-runtime-extensions-nutanix:$$(gojq -r .version dist/metadata.json)
 	kubectl rollout restart deployment cluster-api-runtime-extensions-nutanix
 	kubectl rollout status deployment cluster-api-runtime-extensions-nutanix
+
+
+.PHONY: dev.update-bootstrap-credentials-aws
+dev.update-bootstrap-credentials-aws: export KUBECONFIG := $(KIND_KUBECONFIG)
+dev.update-bootstrap-credentials-aws: export AWS_B64ENCODED_CREDENTIALS := $(shell clusterawsadm bootstrap credentials encode-as-profile)
+dev.update-bootstrap-credentials-aws:
```

nit: maybe this so it reads better?
```suggestion
.PHONY: dev.update-aws-bootstrap-credentials
dev.update-aws-bootstrap-credentials: export KUBECONFIG := $(KIND_KUBECONFIG)
dev.update-aws-bootstrap-credentials: export AWS_B64ENCODED_CREDENTIALS := $(shell clusterawsadm bootstrap credentials encode-as-profile)
dev.update-aws-bootstrap-credentials:
```